NomadicNibbler/foodtruck-fe
# Update/manifest file

## What does this PR do?
- The manifest file was updated to included proper configuration for a PWA
- icons, name, short_name, display settings, scope and start_url

## How should it be tested?

- pull the branch down, run a lighthouse audit for a PWA. The audit should pass without any errors

## What do the future iterations hold?

- N/A

## Linked Issues
